LICENSE.md

```diff
@@ -0,0 +1,21 @@
+# Licenses
+
+## Specification License
+
+Specifications in the repository are subject to the **Community Specification License 1.0** available at [https://github.com/CommunitySpecification/1.0](https://github.com/CommunitySpecification/1.0).
+
+## Source Code License
+
+If source code is included in this repository, or for sample or reference code included in the specification itself, that code is subject to the [Apache-2.0 License](https://www.apache.org/licenses/LICENSE-2.0) unless otherwise designated. In the case of any conflict or confusion within this specification repository between the Community Specification License and the [Apache-2.0 License](https://www.apache.org/licenses/LICENSE-2.0) or other designated license, the terms of the Community Specification License shall apply.
+
+If source code is included in this repository, or for sample or reference code included in the specification itself, that code is subject to the [Apache-2.0 License](https://www.apache.org/licenses/LICENSE-2.0) unless otherwise marked.
+
+In the case of any conflict or confusion within this specification repository between the Community Specification License and the designated source code license, the terms of the Community Specification License shall apply.
+
+# FDC3 Licensing Terms For Past Versions
+
+[Version 1.0 of the FDC3 specification](https://fdc3.finos.org/docs/1.0/fdc3-standard) is licensed under the [FDC3 1.0 Final Specification License](PATENTS-FDC3-1.0.md). 
+
+[FDC3 specification versions 1.1](https://fdc3.finos.org/docs/1.1/fdc3-standard) and  [FDC3 Specification Version 1.2](https://fdc3.finos.org/docs/1.2/fdc3-standard) are subject to the [FINOS IP Policy](https://github.com/finos/community/blob/master/website/static/governance-docs/IP-Policy.pdf), which authorizes implementation of FDC3 specifications without charge, on a RAND basis, subject to the terms of the policy. For details of the IP commitments made by contributors to FDC3, please refer to the policy.
+
```

Comment:
@robmoffat we're missing a reference to the 2.0 version of the standard. 

@scooter99boston I think it should say "1.1, 1.2 and 2.0 are subject to the FINOS IP Policy"

Reply:
@kriswest can you comment on this?

Reply:
They are correct - 1.1, 1.2 and 2.0 are all subject to the FINOS IP policy